### DISCUSSION TITLE
Add text link for PR's on Action Summary

### DISCUSSION THREAD

**Discussion by Mysterious-Dev** [community/community#58269] [Mobile]
This can improve the UX to add a "clickable text" on the action summary (see video below).

https://github.com/community/community/assets/40738104/10849bdb-564c-4b31-b973-89454a247fba

**Comment by iskae** [comment]
Hi @Mysterious-Dev 👋 

Thanks a bunch for reaching out and adding a video of your request ❤️ That helps us a lot 🙇 

I have created an internal tracking issue for your request 👍 I think it makes sense have this clickable, so that we can navigate from the summary view to the PR directly 👍 

Please stay tuned for the future versions of our mobile apps 🙇 

Cheers!
